build_modules/lib/src/kernel_builder.dart

```diff
@@ -265,7 +266,7 @@ Future<void> _addRequestArguments(
   request.inputs.add(Input()
     ..path = '${Uri.file(p.join(sdkDir, sdkKernelPath))}'
     // Sdk updates fully invalidate the build anyways.
-    ..digest = [0]);
+    ..digest = md5.convert(utf8.encode(platform.name)).bytes);
```

Comment:
Should we add this as a getter on DartPlatform with caching? probably doesn't matter much though.

Reply:
Discussed offline - probably not worth it.